lint-release-notes/action.yaml

```diff
@@ -36,9 +36,11 @@ runs:
       id: find-pull-request
       with:
         state: open
+    - run: echo ${{ github.action_path }}
+      shell: bash
     - name: Generate release notes
       id: release-notes-preview
-      uses: open-turo/actions-release/semantic-release@v2
+      uses: ${GITHUB_ACTION_PATH}/../semantic-release
```

Comment:
For this to work we have to add `actions/checkout` it seems, so better to keep it the other way I think